### PR TITLE
prepare 1.11.0

### PR DESCRIPTION
**Pull request by Keruspe**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,23 @@ and this project adheres to [Semantic Versioning](https://book.async.rs/overview
...
 
+# [1.11.0] - 2022-03-17
+
+This release comes with an assortment of small features and fixes.
+
+## Added
+- `TcpListener::into_incoming` to convert a `TcpListener` into a stream of incoming TCP connections
+
+## Removed
+- Internal `extension_trait` macro had been removed and its users rewritten.
```
Comment (yoshuawuyts) [inline CHANGELOG.md:22]:
```suggestion
- The internal `extension_trait` macro had been removed. This drastically improves compile times for `async-std`, but changes the way our documentation is rendered. This is a cosmetic change only, and all existing code should continue to work as it did before.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,23 @@ and this project adheres to [Semantic Versioning](https://book.async.rs/overview
 ## Removed
 ## Changed
 
+# [1.11.0] - 2022-03-17
+
+This release comes with an assortment of small features and fixes.
```
Comment (yoshuawuyts) [inline CHANGELOG.md:16]: I think we can highlight the faster compile times here.
```suggestion
This release improves compile times by up to 55% on initial builds, and up to 75% on recompilation. Additionally we've added a few new APIs and made some tweaks.
```

**Review by yoshuawuyts — APPROVED** [review]
Overall this looks good! -- added a few notes, but nothing which should block a release!

-- excited for this!

**Review by dignifiedquire — APPROVED** [review]
👍

**Pull request merged**